windows/CodePushReactPackage.cs

```diff
@@ -1,25 +1,18 @@
 ﻿using Newtonsoft.Json.Linq;
-using Newtonsoft.Json;
 using ReactNative;
 using ReactNative.Bridge;
 using ReactNative.Modules.Core;
 using ReactNative.UIManager;
 using System;
 using System.Collections.Generic;
-using System.Reflection;
 using System.Threading.Tasks;
-using Windows.ApplicationModel;
-using Windows.Storage;
-using Windows.Web.Http;
+
 
 namespace CodePush.ReactNative
 {
     public sealed class CodePushReactPackage : IReactPackage
     {
         private static CodePushReactPackage CurrentInstance;
-        private static bool NeedToReportRollback = false;
-
```

Comment:
@matthargett I suspect the purpose of this is to post to the CodePush server after a rollback happens, so that users can see a record of the number of rollbacks in the CodePush CLI. If you'd like to file an issue and do a separate PR I am fine with that!